src/coreclr/gc/gc.cpp

```diff
@@ -49541,7 +49687,8 @@ void gc_heap::do_pre_gc()
         settings.condemned_generation,
         total_allocated_since_last_gc,
         (settings.concurrent ? "BGC" : (gc_heap::background_running_p() ? "FGC" : "NGC")),
-        settings.b_state));
+        settings.b_state,
+        n_heaps));
```

Comment:
keep this?

Reply:
I wouldn't keep this since this is a message that's interpreted by the parser for dprintf stress log. to change it would require to make sure it doesn't break that (which we can do) but it doesn't seem urgent to do. so let's not keep this change which mean you can also get rid of the change in gcmsg.inl